test/cases/getUserInfo.ts

```diff
@@ -80,19 +80,18 @@ describe('getUserInfo', () => {
         .catch(err => ("couldn't call endpoint"));
     });
 
-    it('Should get warning data with public ID', async () => {
-        try {
-            const res = await fetch(getbaseURL() + '/api/userInfo?userID=' + await getHash("getuserinfo_warning_0"))
-            
+    it('Should get warning data with public ID', (done: Done) => {
+        fetch(getbaseURL() + '/api/userInfo?publicUserID=' + getHash("getuserinfo_warning_0"))
```

Comment:
I don't think you can remove the await here

Reply:
In every other implementation of `getHash`, it's never awaited, except for here